operations/image/dffml_operations_image/operations.py

```diff
@@ -0,0 +1,42 @@
+from typing import List
+import cv2
+import numpy as np
+
+from dffml.df.base import op
+
+from .definitions import (
+    resized_array,
+    data_array,
+    old_dimensions,
+    new_dimensions,
+)
+
+
+@op(
+    inputs={
+        "data": data_array,
+        "old_dim": old_dimensions,
+        "new_dim": new_dimensions,
+    },
+    outputs={"resized_array": resized_array},
+    conditions=[],
+)
+async def resize(data: np.ndarray, old_dim: List[int], new_dim: List[int]):
+    """
+    Resize array
+    """
+    image = np.array(data).reshape(old_dim)
+    if len(new_dim) == 2:
+        image = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY)
+    else:
+        new_dim = new_dim[:2]
+    if new_dim > old_dim:
+        resized_image = cv2.resize(
```

Comment:
Can we add a comment saying that effect this will have, or better yet write a description in the docstring

Reply:
Okay!